lib/ui.rb

```diff
@@ -1,25 +1,27 @@
+require 'clear_screen_printer'
+
 class Ui
   attr_reader :input, :output
-  CLEAR_SCREEN = "\e[H\e[2J"
 
   def initialize(input, output)
     @input = input
     @output = output
+    @printer = ClearScreenPrinter.new(output)
   end
 
-  def menu(options)
-    output.puts "#{CLEAR_SCREEN} :::Contacts management::: \n\nPlease choose a menu option:\n\n"
+  def display_menu(options)
+    printer.print(" :::Contacts management::: \n\nPlease choose a menu option:\n\n")
     options.each do |number, option|
-      output.puts "#{number} - #{option}"
+      output.puts "#{number} - #{format_for_display(option)}"
     end
     output.print"\n---> "
```

Comment:
I approached it that way in the beginning but then I would have had to change all my Ui tests... So for now I just wrapped the `puts` and `CLEAR_SCREEN` code in an abstraction. 

I can extend the printer to include all print and puts. And for the tests I can then save the string that the printer printed in a variable and assert that it's the correct one. I'll add that to my list and if I have enough time, I'll make that change.